### PR TITLE
support for icons in panel submenu header

### PR DESCRIPTION
**Pull request by sandeepmantha**
# Description

Support for images on the header of the submenu panel. 

Fixes # 18906

## Overview of Changes
- Removed the @Section(Type.MENUPANEL) and replaced with @MenuPanel, that supports img-src.

## Type of change
UI Change

Please delete options that are not relevant.
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

# Additional Notes

Please add any additional notes regarding this pull request here.

### CODE REVIEW

**Comment by openanthem-admin** [comment]
You did it @sandeepmantha!

Thank you for signing the Anthem Opensource Software CLA.

File: nimbus-core/src/main/java/com/antheminc/oss/nimbus/domain/defn/ViewConfig.java [hunk 1]
```diff
@@ -22,6 +22,7 @@
 import java.lang.annotation.Target;
 
 import com.antheminc.oss.nimbus.domain.Event;
+import com.antheminc.oss.nimbus.domain.defn.ViewConfig.Page;
```
Comment (lopezton-anthem) [inline nimbus-core/src/main/java/com/antheminc/oss/nimbus/domain/defn/ViewConfig.java]: Is this import needed?

File: nimbus-ui/nimbusui/src/app/services/layout.service.ts [hunk 2]
```diff
@@ -164,7 +164,7 @@ export class LayoutService {
 
         layoutConfig.params.forEach(param => {
             let attribute: UiAttribute = param.config.uiStyles.attributes;
-            if (attribute.alias === 'Header' || attribute.alias === 'Global-Header' || 
+            if (attribute.alias === 'Header' || attribute.alias === 'Global-Header' ||
```
[the remark this answers is not in the document]
Comment (sandeepmantha) [reply nimbus-ui/nimbusui/src/app/services/layout.service.ts]: I thought I ll refactor just these 2 lines but then realized that in the next few days we will remove this altogether and create new annotations that would replace what we have today. So did not change them to use defined constants.

[the remark this answers is not in the document]
Comment (lopezton-anthem) [reply nimbus-ui/nimbusui/src/app/services/layout.service.ts]: Ok.

**Pull request merged**